modules/nf-core/genomad/endtoend/main.nf

```diff
@@ -0,0 +1,123 @@
+process GENOMAD_ENDTOEND {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::genomad=1.5.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/genomad:1.5.2--pyhdfd78af_0':
+        'biocontainers/genomad:1.5.2--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    path  genomad_db
+    val   score_calibration
```

Comment:
Why is the Val passed here? From the command it doesn't look like it is mandatory?

Reply:
its used in the pre script portion
`def score_calibration_cmd = score_calibration ? '--enable-score-calibration' : ''`

Reply:
yes, but can't this go to ext.args?

Reply:
absolutely :+1: 

@CarsonJM could you explore providing this via `ext.args` rather than as input

Reply:
@FriederikeHanssen @sateeshperi Thank you for the feedback! The reason I have it as a mandatory argument is because this option changes the output files that will be produced (e.g. the optional outputs). But the main reason I have it set this way is that I use it for my stub commands to try and make them match the tool's actual output as close as possible (line 54)
```
    if ( score_calibration_cmd == '--enable-score-calibration' )
```
Is there a way to reference an ext.arg to change the files output by the stub command? If so, I can just use that and replace the input val!

Reply:
In the stub section you are already using task.ext.Args so you should have access. Since the changed output is anyways optional it shouldn't matter

Reply:
Thanks for the clarification, that makes sense!